README.md

```diff
@@ -1,121 +1,105 @@
 
-> [hyper](https://github.com/hyperium/hyper) client and server bindings for [unix domain sockets](https://github.com/tokio-rs/tokio-uds)
+> [Hyper](https://github.com/hyperium/hyper) client and server bindings for [Unix domain sockets](http://github.com/tokio-rs/tokio/tree/master/tokio-uds/)
 
-Hyper is a rock solid [rustlang](https://www.rust-lang.org/) HTTP client and server tool kit. [Unix domain sockets](https://en.wikipedia.org/wiki/Unix_domain_socket) provide
-a mechanism for host-local interprocess communication. Hyperlocal builds on and complements hyper's interfaces for building unix domain socket HTTP clients and servers.
+Hyper is a rock solid [Rust](https://www.rust-lang.org/) HTTP client and server toolkit.
+[Unix domain sockets](https://en.wikipedia.org/wiki/Unix_domain_socket) provide a mechanism
+for host-local interprocess communication. `hyperlocal` builds on and complements Hyper's
+interfaces for building Unix domain socket HTTP clients and servers.
 
-This is useful for exposing simple HTTP interfaces for your Unix daemons in cases where you want to limit access to the current host, in which case, opening and exposing tcp ports is not needed. Examples of unix daemons that provide this kind of host local interface include, [docker](https://docs.docker.com/engine/misc/), a process container manager.
+This is useful for exposing simple HTTP interfaces for your Unix daemons in cases where you
+want to limit access to the current host, in which case, opening and exposing tcp ports is
+not needed. Examples of Unix daemons that provide this kind of host local interface include
+[Docker](https://docs.docker.com/engine/misc/), a process container manager.
 
 
-## install
+## Installation
 
 Add the following to your `Cargo.toml` file
 
 ```toml
 [dependencies]
-hyperlocal = "0.6"
+hyperlocal = "0.7-alpha.1"
 ```
 
-## usage
+## Usage
 
-### servers
+### Servers
 
-A typical server can be built with `hyperlocal::server::Server`
+A typical server can be built with `hyperlocal::server::UnixServerExt`.
 
 ```rust
-extern crate hyper;
-extern crate hyperlocal;
-extern crate futures;
+#![feature(async_await)]
 
+use std::error::Error;
 use std::fs;
-use std::io;
+use std::path::Path;
 
-use hyper::{header, Body, Request, Response};
-use hyper::service::service_fn;
+use hyper::{
+    service::{make_service_fn, service_fn},
+    Body, Response, Server,
+};
+use hyperlocal::server::UnixServerExt;
 
 const PHRASE: &'static str = "It's a Unix system. I know this.";
 
-fn hello(_: Request<Body>) -> impl futures::Future<Item = Response<Body>, Error = io::Error> + Send {
-    futures::future::ok(
-        Response::builder()
-            .header(header::CONTENT_TYPE, "text/plain")
-            .header(header::CONTENT_LENGTH, PHRASE.len() as u64)
-            .body(PHRASE.into())
-            .expect("failed to create response")
-    )
-}
+#[hyper::rt::main]
```

Comment:
Where is this hyper::rt::main attribute defined?

Reply:
It's a re-export of `tokio::main`, here: https://github.com/hyperium/hyper/blob/f0478c62677a953aae84aa8d976bec1d28ef21b1/src/rt.rs#L14

Reply:
Seem's like it's been removed.